lib/ansible/modules/network/checkpoint/cp_mgmt_mds_facts.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Ansible module to manage CheckPoint Firewall (c) 2019
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = """
+---
+module: cp_mgmt_mds_facts
+short_description: Get mds objects facts on Checkpoint over Web Services API
```

Comment:
@chkp-amiads can u plz expand the abbreviated form once here in description, so that user might get to know the actual operation

Reply:
Do you mean to explain what is mds object he should expect to see?

Reply:
@chkp-amiads Yes, can u change the description to: `Get Multi-Domain Server (mds) objects facts on Checkpoint over Web Services API`